homeassistant/components/weather/__init__.py

```diff
@@ -103,6 +105,16 @@ def visibility(self):
         """Return the visibility."""
         return None
 
+    @property
+    def rain_forecast(self):
```

Comment:
https://developers.home-assistant.io/docs/en/entity_index.html#changing-the-entity-model